src/modules/launcher/PowerLauncher/ViewModel/MainViewModel.cs

```diff
@@ -451,37 +431,37 @@ public Visibility MainWindowVisibility
             }
         }
 
-        public ICommand IgnoreCommand { get; set; }
+        public ICommand IgnoreCommand { get; private set; }
 
-        public ICommand EscCommand { get; set; }
+        public ICommand EscCommand { get; private set; }
 
-        public ICommand SelectNextItemCommand { get; set; }
+        public ICommand SelectNextItemCommand { get; private set; }
 
-        public ICommand SelectPrevItemCommand { get; set; }
+        public ICommand SelectPrevItemCommand { get; private set; }
 
-        public ICommand SelectNextContextMenuItemCommand { get; set; }
+        public ICommand SelectNextContextMenuItemCommand { get; private set; }
 
-        public ICommand SelectPreviousContextMenuItemCommand { get; set; }
+        public ICommand SelectPreviousContextMenuItemCommand { get; private set; }
 
-        public ICommand SelectNextTabItemCommand { get; set; }
+        public ICommand SelectNextTabItemCommand { get; private set; }
 
-        public ICommand SelectPrevTabItemCommand { get; set; }
+        public ICommand SelectPrevTabItemCommand { get; private set; }
 
-        public ICommand SelectNextPageCommand { get; set; }
+        public ICommand SelectNextPageCommand { get; private set; }
 
-        public ICommand SelectPrevPageCommand { get; set; }
+        public ICommand SelectPrevPageCommand { get; private set; }
 
-        public ICommand SelectFirstResultCommand { get; set; }
+        public ICommand SelectFirstResultCommand { get; private set; }
 
-        public ICommand LoadContextMenuCommand { get; set; }
+        public ICommand LoadContextMenuCommand { get; private set; }
 
-        public ICommand LoadHistoryCommand { get; set; }
+        public ICommand LoadHistoryCommand { get; private set; }
 
-        public ICommand OpenResultWithKeyboardCommand { get; set; }
+        public ICommand OpenResultWithKeyboardCommand { get; private set; }
 
-        public ICommand OpenResultWithMouseCommand { get; set; }
+        public ICommand OpenResultWithMouseCommand { get; private set; }
 
-        public ICommand ClearQueryCommand { get; set; }
+        public ICommand ClearQueryCommand { get; private set; }
 
```

Comment:
@davidegiacometti I was wondering about these the other day - are we actually using all of these commands? My gut feeling is that this is all Wox stuff?

Reply:
Will check later!